src/rars/tools/ConversionTool.java

```diff
@@ -0,0 +1,242 @@
+package rars.tools;
+
+import javax.swing.*;
+import javax.swing.event.DocumentEvent;
+import javax.swing.event.DocumentListener;
+import java.awt.*;
+
+public class ConversionTool {
+    private boolean updating = false;
+
+    /**
+     * Creates a panel containing the conversion tool.
+     * 
+     * @return A JPanel containing the conversion tool.
+     */
+
+    public JPanel createConversionToolPanel() {
+        JPanel panel = new JPanel();
+        panel.setLayout(new FlowLayout());
+
+        // Create labels and text fields for each format
+        JLabel decimalLabel = new JLabel("Decimal:");
+        JTextField decimalField = new JTextField(5);
+        decimalField.setToolTipText("Enter a decimal number (e.g., 97)");
+
+        JLabel hexLabel = new JLabel("Hex:");
```

Comment:
I would prefer something more consistent in the labels. Either Decimal/Hexadecimal/Binary or Dec/Hex/Bin